frontend/src/hocs/withMapboxToken/MapboxToken.ts

```diff
@@ -34,7 +35,16 @@ export class MapboxToken {
    * only be fetched once per session.)
    */
   public static async get(): Promise<string> {
-    if (MapboxToken.token == null) {
+    if (
+      !this.isItRunningLocal() &&
+      SessionInfo.current.userMapboxToken === ""
+    ) {
+      throw new Error(
+        "To use this you'll need a Mapbox access token. Please add it to your config."
+      )
+    }
+
```

Comment:
I might be misunderstanding, but it looks like this means that an app's users will see errors in production that the app's developer will not see during development. 

That's a pretty bad experience. Why not show the error message always?

Reply:
Cool if we bring this up in the eng meeting today? I'm concerned this is the wrong approach; if there's an issue that will appear in a deployed app, we should *always* surface that issue to the developer before they deploy it, IMO.

Reply:
Sure, that makes a lot of sense